sdk/spring/tests.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+jobs:
+    - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+      parameters:
+        TimeoutInMinutes: 240
+        MaxParallel: 2
+        ServiceDirectory: spring
+        Matrix:
+          Linux - Java 8 (AzureCloud):
+            OSVmImage: 'ubuntu-18.04'
+            JavaVersion: '1.8'
+            DisplayName: 'Linux - Java 8'
+            SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+            AZURE_TEST_HTTP_CLIENTS: netty
+        TestStepMavenInputs:
+          options: '-Dmaven.wagon.http.pool=false $(DefaultOptions) -Dmaven.javadoc.skip=true -Drevapi.skip=true -DskipSpringITs=false -pl $(ProjectList)'
```

Comment:
I'm happy to see this PR was updated to pass skipSpringITs into Maven instead of using an environment variable.